pawssums_frontend/src/App.jsx

```diff
@@ -17,11 +17,27 @@ import {
 
 function App() {
   const positionSTL = {lat: 38.62 ,lng:-90.19};
+  const [open, setOpen] = useState(false);
+
   return (
     <>
     <APIProvider apiKey="AIzaSyBB-kFmMyRJ7xPlGzMxPHfYVvtzSTUDDsg">
       <div style = {{height: "60vh", width: "100vh"}}>
-          <Map zoom={9} center={positionSTL}></Map>
```

Comment:
Can you delete this line of code, because right now it's displaying 2 maps on the home page